geonode_mapstore_client/client/js/components/MainErrorView/MainErrorView.jsx

```diff
@@ -11,14 +11,15 @@ import FaIcon from '@js/components/FaIcon';
 import Message from '@mapstore/framework/components/I18N/Message';
 
 function MainErrorView({
```

Comment:
Let's change the name of this component if it has been used as generic event view for different use case.
MainEventView ?